.github/workflows/pull-request.yaml

```diff
@@ -22,25 +22,21 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - run: make deps
+    - run: make install
     - run: make test
+    # Run coverage on one of the builds, doesn't matter which
+    - if: ${{ matrix.python-version == '3.12' }}
+      run: make cov-xml
+    - if: ${{ matrix.python-version == '3.12' }}
```

Comment:
I don't think you want `if: always` because it's only running on one branch of the matrix. And anyway, if `make cov-xml` fails, how can coverage report?

Reply:
Actually, should it be...?

 `- if: ${{  !cancelled() && matrix.python-version == '3.12' }}`

https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions

`make cov-xml` will fail if the coverage is below report requirements but will still report the coverage. We want this step to run regardless so that the coverage report is added to the Pull Request.

Here is an example of the behavior: https://github.com/posit-dev/posit-sdk-py/pull/67

Reply:
Ah, got it. Thanks for the demonstration. 

Would it be better to just remove `fail_under = 80` from .coveragerc? I'm not sure it's helping us more than having the coverage report on the PR.

Reply:
Either works for me.